### PR TITLE
Change field name.

### PR DESCRIPTION
**Pull request by matgrioni**

### CODE REVIEW

File: actions/feedback-service.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ function update(feedbackClass, cloudant, dbName, minrpi, maxrpi, activities) {
           selector: {
             '$and': [
               {
-                'rpi': {
+                'signDiff': {
                 '$lt': minrpi,
```
Comment (LanJosh) [inline actions/feedback-service.js:90]: Shouldn't the change be on this line from the parameter value minrpi to signDiff if the call from the frontend uses signDiff as field name now?

Comment (matgrioni) [reply actions/feedback-service.js:90]: The discovery function will return a value called max and min rpi, but they are actually signDiff values. The result stored in cloudant has an rpi field and a signDiff field. So by changing the search term from rpi to signDiff we are just changing the logic to filter on signDiff.

Comment (matgrioni) [reply actions/feedback-service.js:90]: The minrpi and maxrpi values are basically minsigndiff and maxsigndiff

**Review by LanJosh — APPROVED** [review]

File: actions/feedback-service.js (hunk 1 above)
Comment (LanJosh) [reply actions/feedback-service.js:90]: Are we changing the name from maxrpi and minrpi in a later PR?

Comment (LanJosh) [reply actions/feedback-service.js:90]: well actually the class ends on Thursday so I guess there not a ton of need to do so.

**Pull request merged**